engine-helpers/src/index.ts

```diff
@@ -297,7 +297,7 @@ export interface LoadFitsLayerOptions {
 
   /** Whether to seek the view to the positon of the FITS file on the sky,
    * if/when it successfully loads. */
-  gotoTarget: boolean;
+  goto: boolean;
```

Comment:
Note that renaming these fields is an API break, although in this case I think it is acceptable.

Reply:
It is already deprecated layer options though, so this API break has very low value. I'll just revert it and have the new name for the new layer option class